doc/releases/changelog-dev.md

```diff
@@ -187,6 +187,33 @@
 
 <h3>Breaking changes 💔</h3>
 
+* The `qml.HilbertSchmidt` and `qml.LocalHilbertSchmidt` templates have been updated and their UI has been remarkably simplified. 
+  They now accept a list of operations instead of a `qml.tape.QuantumScript` of operations and quantum functions with separate parameters and wires.
```

Comment:
```suggestion
  They now accept an operation or a list of operations instead of a `qml.tape.QuantumScript` of operations and quantum functions with separate parameters and wires.
```